bot/code_coverage_bot/taskcluster.py

```diff
@@ -87,6 +87,7 @@ def download_artifact(artifact_path, task_id, artifact_name):
     url = queue.buildUrl("getLatestArtifact", task_id, artifact_name)
     logger.debug("Downloading artifact", url=url)
 
+    @retry(wait=wait_fixed(30), stop=stop_after_attempt(5))
```

Comment:
What would be the defaults?

Reply:
It would retry forever without waiting without the arguments. The numbers were chossen was they were the defaults used in retry function.

Reply:
OK, sounds good!